smart-contracts/Makefile

```diff
@@ -0,0 +1,33 @@
+# Makefile
+
+SHELL:=/bin/bash
+
+# Run slither over entire directory
+# `make slither`
+slither:
+	python3 -m venv venv
+	source ./venv/bin/activate
+	pip3 install -r requirements.txt
+	yarn
+	# Slither . doesn't return a 0 even when it works, so the makefiles always failing. 
+	# This is a hacky way to contain that error, and if slither does fail for genuine reasons
+	# it will have been echo'd for debugging purposes anyway. 
+	slither . || echo 
```

Comment:
Doesn't really matter, but I've usually seen true instead of echo for things like this.

Reply:
Fair enough - I like your way better, updated :)